kedro-docker/RELEASE.md

```diff
@@ -6,6 +6,9 @@
 
 ## Breaking changes to the API
 
+# Release 0.5.0
+## Major features and improvements
+* Add compatibility with `kedro` 0.19.x
```

Comment:
Technically I think even the older version works with 0.19. The only changes is we assume 0.19.x template so requirements is moved 1 level up. I will say this is a breaking changes too, and add a note that if people need to use this for old projects, they should either use a older `kedro-docker` version or move the requirements file up.